zenoh/src/publication.rs

```diff
@@ -198,6 +202,35 @@ use std::pin::Pin;
 use std::task::{Context, Poll};
 use zenoh_result::Error;
 
+#[zenoh_macros::unstable]
+#[derive(Clone)]
+pub enum PublisherRef<'a, 'b: 'a> {
```

Comment:
Are the 2 lifetimes really necessary? Everything passes a single lifetime from then on

Reply:
Probably not. Removed them.